packages/node_modules/@webex/plugin-meetings/src/statsAnalyzer/index.js

```diff
@@ -1,10 +1,18 @@
+
 import EventsScope from '../common/events/events-scope';
-import {
-  DEFAULT_GET_STATS_FILTER, EVENT_TRIGGERS, PEER_CONNECTION_STATE, STATS
-} from '../constants';
+import {EVENT_TRIGGERS,
+  DEFAULT_GET_STATS_FILTER, PEER_CONNECTION_STATE, STATS, MQA_INTEVAL} from '../constants';
+import mqaConfig from '../mediaQualityMetrics/config';
```

Comment:
Since this isn't really a config, could we call it something like `mqaData`?

Reply:
done